saleor/order/utils.py

```diff
@@ -13,10 +13,68 @@
 from ..dashboard.order.utils import get_voucher_discount_for_order
 from ..discount.models import NotApplicable
 from ..order import FulfillmentStatus, OrderStatus
-from ..order.models import OrderLine
+from ..order.models import Fulfillment, FulfillmentLine, Order, OrderLine
 from ..payment import ChargeStatus
 from ..payment.utils import gateway_refund, gateway_void
-from ..product.utils import allocate_stock, deallocate_stock, increase_stock
+from ..product.utils import (
+    allocate_stock, deallocate_stock, decrease_stock, increase_stock)
+from ..product.utils.digital_products import (
+    get_default_digital_content_settings)
+
+
+def order_line_needs_automatic_fulfillment(line: OrderLine) -> bool:
+    """Check if given line is digital and should be automatically fulfilled"""
+    digital_content_settings = get_default_digital_content_settings()
+    default_automatic_fulfillment = (
+        digital_content_settings['automatic_fulfillment'])
+    content = line.variant.digital_content
+    if default_automatic_fulfillment and content.use_default_settings:
+        return True
+    if content.automatic_fulfillment:
+        return True
+    return False
+
+
+def order_needs_automatic_fullfilment(order: Order) -> bool:
+    """Check if order has digital products which should be automatically
+    fulfilled"""
+    digital_lines = [line for line in order if line.is_digital]
+    if not digital_lines:
+        return False
+    for line in digital_lines:
```

Comment:
I am not sure if I like this loop. I would consider to make a `property` or `manager` for an `OrderLine` model and then one could use it as a `filter` condition.
But it seems to work, so if there are no big benefits from change, it's fine.

Reply:
I don't want to put it inside the `models` file, keeping in mind how fast `models` file can grow. 
We can ask Marcin also. Is it fine for you?

Reply:
Yes. Maybe manager could do the job then.